src/components/ThemeProvider.tsx

```diff
@@ -22,7 +22,18 @@ function ThemeProvider({children, theme: staticThemePreference}: ThemeProviderPr
     const theme = useMemo(() => themes[debouncedTheme], [debouncedTheme]);
 
     useEffect(() => {
+        /*
+         * For static themes we don't want to apply the autofill input style globally
+         * SignInPageLayout uses static theme and handles this differently.
+         */
+
```

Comment:
Small nitpick but I don't think the style guide calls for a blank line here? It kinda looks odd to me.

```suggestion
```